### PR TITLE
Rails gyn last blog post

### PR DESCRIPTION
**Pull request by branquinhoaa**
Submit Rails Girls Last Blog Post

### CODE REVIEW

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: RailsGyn come back to reality.
+layout: post
+created_at: Fri September 22 2017
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: Hi, thanks for the PR. Your blogpost is scheduled to be published on Thursday September 27, 2017. Can you please update the date everywhere?

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<img src="/img/blog/2017/icons.png" alt="rgsoc">
+<div align="center" div class="image-credits">Design by Juliana</div>
+
+## What were our main achievements in this summer?
+
+We started the summer of code working on beginner friendly issues, however, on the very first days, one of our friends from TeamsApp asked for help in a milestone to automate the conference process to offer conference tickets to teams. It was a big change, with a lot of issues and a lot of models builds and refactors. She even though that it wasn’t possible to build for this summer, but we did it! We build almost all the related things to this milestone conclusion and when we saw it on production was proud.
+It was really surrounding construct such a complex issue in a really new project for us. We had to learn a lot in just some days, but worth every second when we saw everything working.
+That is was a challenge do with you, it makes you find out how far you can go.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: This sentence 'That is was a challenge do with you, it makes you find out how far you can go' looks bit odd. Can you rephrase it?

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+After finish the conference milestone, we have decided to start a community milestone and we still are working on it. Even if we did not finish it until september/30 we will finish it!
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: Change 'finish' -> 'finishing', 'start a community'->'start **on** a community'

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+<img src="/img/blog/2017/we.jpg" alt="us">
+<div align="center" div class="image-credits">By: Juliana's mother</div>
+
+## What will we miss?
+* We will miss all the excellent developers from team's app who worked closely with us in this summer: Carsten, Max, Maria, Maud...
+* We will miss Ines :\ our conversations were always the best ones and we don’t want to stop it.
+* We will miss sharing in all the social media that we work on the RGSoC.
+* We will miss pair working in nice coffee places in our city.
+
+## What do we learn?
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: Change 'do'->'did'

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+Well, this summer was a great learning experience for us, our mentors were fantastic with your code reviews, it makes us grow up a lot. During this summer we can learn more about tests with RSpec, using shared examples, many factories, finite state machine concept, some Ruby methods, code best practices applying DRY,  metaprogramming, service objects, work with Continuous Integration, Git workflow and many ruby facts that we happily shared on our [blog](https://medium.com/@railsgirlsgyn).
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: This line: 'our mentors were fantastic with your code reviews, it makes us grow up a lot looks bit odd too.' Can you change/rephrase it as well?

File: blog/_posts/2017-09-8-railsgyn-lastblogpost.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## What more?
+Besides knowledge, the program brought us a lot of local visibility. Due to the program, one of the local companies has contacted us to offer us support to promote events. Also, this same company calls us for a job interview. We are planning to promote a Rails Girls event in our city, and this company will give us space and some coaches to assist us in the Rails Girls workshop. We are very happy for all the opportunities that the project has given us, we will miss everything.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-8-railsgyn-lastblogpost.md]: In 'contacted us to offer us support', remove one '**us**'. How about: '.has contacted us to offer support **for promoting** events' ? 
Change 'calls us'->'has called us'

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by branquinhoaa** [comment]
Thanks for your review @nerdyvaishali! I think now it is ok!

**Review by v-thakkar — APPROVED** [review]
